core-web/libs/portlets/dot-content-drive/portlet/src/lib/components/dot-content-drive-toolbar/components/dot-content-drive-content-type-field/dot-content-drive-content-type-field.component.ts

```diff
@@ -0,0 +1,130 @@
+import { patchState, signalState } from '@ngrx/signals';
+import { of, Subject } from 'rxjs';
+
+import {
+    ChangeDetectionStrategy,
+    Component,
+    computed,
+    effect,
+    inject,
+    signal
+} from '@angular/core';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { FormsModule } from '@angular/forms';
+
+import { MultiSelectFilterEvent, MultiSelectModule } from 'primeng/multiselect';
+import { SkeletonModule } from 'primeng/skeleton';
+
+import { catchError, debounceTime, distinctUntilChanged, switchMap, tap } from 'rxjs/operators';
+
+import { DotContentTypeService } from '@dotcms/data-access';
+import { DotCMSContentType } from '@dotcms/dotcms-models';
+import { DotMessagePipe } from '@dotcms/ui';
+
+import { DEBOUNCE_TIME, MAP_NUMBERS_TO_BASE_TYPES } from '../../../../shared/constants';
+import { BASE_TYPES } from '../../../../shared/models';
+import { DotContentDriveStore } from '../../../../store/dot-content-drive.store';
```

Comment:
Should we add a path to the `tsconfig`?

Reply:
I can handle this on my PR

Reply:
It is not possible, since we have `enforce-module-boundaries` rule to avoid circular dependencies, we cannot use absolute paths within the same library, if the file comes from the same module, we should use relative pathing